ods-devenv/scripts/deploy.sh

```diff
@@ -0,0 +1,1574 @@
+        echo "Deleting repository opendevstack/${repository} on http://${public_hostname}:${atlassian_bitbucket_port}."
+        curl -X DELETE --user openshift:openshift "http://${public_hostname}:${atlassian_bitbucket_port}/rest/api/1.0/projects/opendevstack/repos/${repository}"
+    done
+}
+
+#######################################
+# Makes use of ods-core/ods-setup-repos.sh to clone ods repositories from github
+# and push them to the local BitBucket instance.
+# Globals:
+#   atlassian_bitbucket_port
+# Arguments:
+#   n/a
+# Returns:
+#   None
+#######################################
+function initialise_ods_repositories() {
+    local opendevstack_dir="/home/${USER}/opendevstack"
+
+    mkdir -p "${opendevstack_dir}"
+    pushd "${opendevstack_dir}"
+    curl -LO https://raw.githubusercontent.com/opendevstack/ods-core/${ods_git_ref}/ods-setup/repos.sh
+    chmod u+x ./repos.sh
+    ./repos.sh --init --confirm --source-git-ref "${ods_git_ref}" --target-git-ref "${ods_git_ref}" --bitbucket "http://openshift:openshift@${atlassian_bitbucket_host}:${atlassian_bitbucket_port_internal}" --verbose
+    ./repos.sh --sync --bitbucket "http://openshift:openshift@${atlassian_bitbucket_host}:${atlassian_bitbucket_port_internal}" --source-git-ref "${ods_git_ref}" --target-git-ref "${ods_git_ref}" --confirm
+    popd
+}
+
+function inspect_bitbucket_ip() {
+    atlassian_bitbucket_ip=$(docker inspect -f '{{.NetworkSettings.IPAddress}}' "${atlassian_bitbucket_container_name}")
+}
+
+function inspect_jira_ip() {
+    atlassian_jira_ip=$(docker inspect -f '{{.NetworkSettings.IPAddress}}' "${atlassian_jira_container_name}")
+}
+
+function inspect_crowd_ip() {
+    atlassian_crowd_ip=$(docker container inspect --format '{{.NetworkSettings.IPAddress}}' "${atlassian_crowd_container_name}")
+}
+
+#######################################
+# Creates a config file and uploads it into ods-configuration repository on
+# the local BitBucket instance.
+# Globals:
+#   atlassian_bitbucket_ip
+#   atlassian_bitbucket_port_internal
+#   atlassian_jira_ip
+#   atlassian_jira_port_internal
+#   public_hostname
+# Arguments:
+#   n/a
+# Returns:
+#   None
+#######################################
+function create_configuration() {
+    echo "create configuration"
+    pwd
+    ods-setup/config.sh --verbose --bitbucket "http://openshift:openshift@${atlassian_bitbucket_host}:${atlassian_bitbucket_port_internal}"
+    pushd ../ods-configuration
+    git init
+    # keep ods-core.env.sample as a reference
+    # echo "ods-core.env.sample" > .gitignore
+
+    if ! git remote | grep origin; then
+        git remote add origin "http://openshift:openshift@${atlassian_bitbucket_host}:${atlassian_bitbucket_port_internal}/scm/opendevstack/ods-configuration.git"
+    fi
+    git add -- .
+    git commit -m "initial commit"
+    git push --set-upstream origin master
+    # base64('changeit') -> Y2hhbmdlbWUK
+    sed -i "s|ODS_GIT_REF=.*$|ODS_GIT_REF=${ods_git_ref}|" ods-core.env
```

Comment:
There is already a script in the repo which creates the same configuration. See https://github.com/opendevstack/ods-core/blob/master/tests/scripts/create-env-from-local-cluster.sh

Please reuse and improve (if needed) the existing one